Surprise.py

```diff
@@ -0,0 +1,18 @@
+def print_surprise_decorator(func):
+    """
+    Print surprise instead of the original print of the function
+    :param func: The function to decorate
+    :return: decorator
+    """
+    def inner(*args):
+        print("surprise")
+    return inner
+
+
+@print_surprise_decorator
+def print_hello():
+    print("Hello")
+
+
+if __name__ == '__main__':
+    print_hello()
```

Comment:
This should not be here.
Don’t use git add -a ever, use git add -p
https://git-scm.com/docs/git-add